index.rst

```diff
@@ -44,14 +44,579 @@
+-----------------------------------------------------------
+
+This working group was not able to find any documented strategy on how the commissioning and diagnostic clusters are to be used during commissioning and the survey.
+High-level descriptions exist from early in the project, however they are not sufficient to build out the system and do not take into account much of the as-built software and hardware capabilities.
+Although beyond the scope of this working group, it is strongly suggested that a strategy be developed that identifies and documents the use-cases, specifically in regards to the differences between how calibration and on-sky data is handled.
+Currently, the camera diagnostic cluster hardware is on the summit but not being used, largely in part due to a lack of definition of it's use-cases and how it is to interact within the global data analysis workflow of the Rubin Observatory, including whether or not DM tooling must be supported.
+
+
+Camera Visualization Tool Functionality Limitations for General Commissioning
+-----------------------------------------------------------------------------
+
+The Camera Visualization Tool, once augmented with the new specifications, will be sufficient for on-the-fly applications but will not be able to satisfy many of the more general commissioning use-cases. 
+The most obvious example is the ability to display and interact with full-frame images once they arrive at NCSA.
+Because the committee was formed to look only at on-the-fly analyses, the following specifications are out of scope, however, for the general commissioning effort to be successful the following functionalities will need to be implemented, or covered by a different suite of tooling.
+
+.. note::
+
+   The items presented here do *not* form a complete set of specifications for general commissioning.
+   They account for merely a subset that we identified as not specifically required for on-the-fly analysis at the mountain top.
+   If the capabilites were in place, then the on-the-fly users would certainly take advantage of them.
+
+FAFF-REQ-014
+^^^^^^^^^^^^
+
+**Specification:** The camera visualization tool shall support being deployed in places where historical data is available.
+
+**Rationale:** This will need to be available to people using the `RSP`_ and commissioning cluster.
+The general commissioning / SV use case is to be able to examine aspects of image quality that cross detector boundaries (e.g., stray and scattered light, satellite trails, pervasive issues across detectors) for which full focal plane visualization is critical. 
+Of course, these studies will involve looking at images that date back in time, and therefore will need to be executed from the RSP (or other processing center).
+This could be useful even for summit operations if it allows display of historic images (for comparison with new images).
+The historical data on the summit is currently limited to 30 days.
+
+**Priority: 1**
```

Comment:
We did not define what Priority = 1 means. Perhaps we should remove the priority field here, since these are requirements that are beyond the current scope, or else define.

Reply:
That's a good point. 
I'll remove them from the report as it is out-of-scope. However, I'll just comment them out and leave the raw text so we can look at what we decided later..